tools/Dockerfile.CI

```diff
@@ -0,0 +1,26 @@
+FROM timbru31/java-node:11-alpine
```

Comment:
I'm curious what this import is, what this file is, and how it's different from alxrsngrtn/arcs-travis-build:latest

I found https://hub.docker.com/r/alxrsngrtn/arcs-travis-build but I can't figure out how to see the contents of the Dockerfile...

Reply:
I think this allows us to see the 'history' of the image (i.e. what's been done to it), but nothing of the contents.

https://hub.docker.com/layers/alxrsngrtn/arcs-travis-build/latest/images/sha256-36cf23c5e93b37faaff3b25c8bac693a2b5d4d517a537adaf3159bc35d9d0851

Ideally this would be far easier to inspect and would not be a binary blob that we're just trusting (we are likely running it's contents on corporate machines which I think will need some review by security).

Reply:
> I'm curious what this import is, what this file is, and how it's different from alxrsngrtn/arcs-travis-build:latest
That's the base image. It combines nodejs (11) and java 8 (open jdk). It's from some developer who seems to keep the combination maintained. 
As a rule of thumb, if you prepend `https://hub.docker.com/repository/docker/` to the name of the image (without the tag), you'll get the url of the image on dockerhub: https://hub.docker.com/repository/docker/timbru31/java-node

Reply:
I'm not sure of all the best-partices for hosting images in docker hub. At least, that's not where I've put energy into this build so far. 
I think a good next-step is to get this image hosted in an arcs-specific account, then invest the time to get it well documented. 

@cromwellian mentioned also something about using bazel to build / manage docker image creation. I'll investigate that later (maybe we'll have an independent CI rule for building the CI rule docker image 😄 )

Reply:
I'm on board with submitting this using your personal account for now, so long as there's a path forward to standardising it somehow. Thanks for all the investigation!